### DISCUSSION TITLE
Looking up available worker count from queue side

### DISCUSSION THREAD

**Discussion by bcomnes** [taskforcesh/bullmq#2460] [Q&A] [answered]
What is the best way to inspect or monitor the number of connected and available workers in bullMQ? The use case for this data:

- I would like to set up a monitor on the queuing side. After a job is scheduled, I would have some kind of monitor that would make sure a worker is connected to the queue to process the job. The idea here is that the worker service could scale to 0 when there is no work. When work is queued, the queue would "kick" the worker service to wake up a process. 

Additionally, monitoring or polling queue size and worker count could be useful for tuning my autoscaling primitives. 

Any ideas or pointers appreciated!

**Comment by manast** [comment, marked as answer]
You can use getWorkers: https://api.docs.bullmq.io/classes/v5.Queue.html#getWorkers

**Reply by bcomnes** [reply]
Perfect! Missed this, thank you.
